internal/pkg/vm/actor/builtin/power/power.go

```diff
@@ -150,8 +150,16 @@ var Errors = map[uint8]error{
 	ErrDuplicateEntry:       errors.NewCodedRevertError(ErrDuplicateEntry, "duplicate create power table entry attempt"),
 }
 
+// Dragons: shouldnt this be calling the initacor to create an actor?
```

Comment:
I'm not sure it even should be in charge of creating a miner actor.  It seems like the miner actor's constructor should call a `createEntry` method on the power actor instead: https://github.com/filecoin-project/specs/issues/635

If miner creation does stay here then yes, it should be calling init (the PR introducing power actor landed before init actually did actor creation)